docs/rst/user_manual/configuration.rst

```diff
@@ -48,26 +54,86 @@ Each Topic is determined by its entries ``name``, ``type`` and ``keyed``, with o
 or not. See Topic section for further information about the topic.
 
-.. todo:
+.. note::
+
+    Tags ``allowlist``, ``blocklist`` and ``builtin-topics`` must be at yaml base level (it must not be inside any
+    other tag).
+
+Allow topic list (``allowlist``)
+--------------------------------
+This is the list of topics that |ddsrouter| will forward, i.e. the data published under the topics defined in
+the ``allowlist`` will be relayed by |ddsrouter|.
+
+
+Block topic list (``blocklist``)
+--------------------------------
+This is the list of topics that the |ddsrouter| will block, that is, all data published under the topics defined in
+the ``blocklist`` will be discarded by the |ddsrouter| and therefore will not be relayed.
+
+This list takes precedence over the ``allowlist``.
+If there is a topic defined in the ``allowlist`` and in the ``blocklist``, the ``blocklist`` takes precedence, causing
+the data of this topic to be discarded.
+
+
+Builtin topics list (``builtin-topics``)
+----------------------------------------
+
+|ddsrouter| has a mechanism to automatically detect which topics are being used in a DDS network.
+By automatically detecting these topics, a |ddsrouter| creates internal DDS :term:`Writers<DataWriter>`
+and :term:`Readers<DataReader>` for each topic and for each Participant in order to relay the data published on each
+discovered topic.
+
+The discovery phase of the network topics can be accelerated by using the builtin topic list (``builtin-topics``).
+By defining topics in this list, the DDS router will create the DataWriters and DataReaders for these topics without
+the need to have previously discovered them.
+In this way, the initialization phase mentioned above is omitted and the application launching efficiency is improved.
+
+Examples of usage
+-----------------
+
+The following is an example of how to use the ``allowlist``, ``blocklist`` and ``builtin-topics`` configurations to
+setup the |ddsrouter| filtering rules.
 
-    Add link to topic page when created
+Dynamic topic discovery example
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-In the following configuration example, the |ddsrouter| will transmit the topic ``rt/chatter`` (default ROS 2 topic for
-``talker`` and ``listener``) with type name ``std_msgs::msg::dds_::String_``.
-It also will transmit the topic ``HelloWorldTopic`` (default FastDDS topic for ``HelloWorldExample``)
-with type name ``HelloWorld``.
+This example shows how the |ddsrouter| is initially configured to forward the ``rt/chatter`` topic (default ROS 2
+topic for ``talker`` and ``listener``) with type name ``std_msgs::msg::dds_::String_``, while the rest of the
+topics in the DDS network are expected to be dynamically discovered.
+Additionally, a rule is defined to block only the ``HelloWorldTopic`` topic.
 
 .. code-block:: yaml
 
-    allowlist:
+    blocklist:
+      - name: "HelloWorldTopic"
```

Comment:
```suggestion
      - name: *
```

Reply:
Just a suggestion in order to include wildcard expressions in the example, maybe it is more interesting for an user to block all topics of a certain type.